lib/twilio-ruby/framework/twilio_response.rb

```diff
@@ -2,14 +2,16 @@ module Twilio
   class TwilioResponse
     attr_accessor :status_code, :body
 
+    # @deprecated Use 'Twilio::Response' instead.
     def initialize(status_code, body)
-      @status_code = status_code
-      body = '{}' if !body || body.empty?
-      @body = JSON.parse(body)
+      warn "'Twilio::TwilioResponse' has been deprecated. Use 'Twilio::Response' instead."
```

Comment:
These classes are pretty much identical. can we just alias TwilioResponse -> Response so that we dont have deprecate anything and users wont have to change their code if they dont want to? Not sure if its possible.

Reply:
Nope.